jade_reader/jade_reader/WindowManager.cpp

```diff
@@ -7,6 +7,7 @@
 WindowManager::WindowManager() {
   firstPageWidget = new UserLoginWindow;
   secondPageWidget = new UserSignUpWindow;
```

Comment:
LoginPageWidget and SignUpPageWidget or something similar might be better names for the QWidget variables.